test/autogen/agentchat/test_responsive_agent.py

```diff
@@ -1,21 +1,21 @@
 import pytest
-from flaml.autogen.agentchat import ResponsiveAgent
+from flaml.autogen.agentchat import ConversableAgent
```

Comment:
The file name should be changed from "test_responsive_agent.py" to "test_conversable_agent.py"?

Reply:
renamed